src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -96,19 +96,6 @@ public unsafe ReadOnlySpan(void* pointer, int length)
             _length = length;
         }
 
-        /// <summary>
-        /// Create a new read-only span over a portion of a regular managed object. This can be useful
-        /// if part of a managed object represents a "fixed array." This is dangerous because neither the
-        /// <paramref name="length"/> is checked, nor <paramref name="obj"/> being null, nor the fact that
-        /// "rawPointer" actually lies within <paramref name="obj"/>.
-        /// </summary>
-        /// <param name="obj">The managed object that contains the data to span over.</param>
-        /// <param name="objectData">A reference to data within that object.</param>
-        /// <param name="length">The number of <typeparamref name="T"/> elements the memory contains.</param>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public static ReadOnlySpan<T> DangerousCreate(object obj, ref T objectData, int length) => new ReadOnlySpan<T>(ref objectData, length);
```

Comment:
At least from my experience, it's easier to stage changes like this across coreclr/corefx if when changing a method's name/location, you first add the new method in coreclr, let that propagate to corefx and fix everything up, and then go back and delete the old method.  Otherwise, there's a period of time where coreclr and corefx don't work together, and if there's any other issue happening at the same time, you may not be able to fix things quickly.